pmm-app/tests/DbaaS/verifyDBaaSPXCCluster_test.js

```diff
@@ -1,66 +1,119 @@
+    await dbaasPage.createClusterBasicOptions(clusterName, pxc_cluster_name, 'MySQL');
+    I.click(dbaasPage.tabs.dbClusterTab.createClusterButton);
+    await dbaasPage.seeErrorForAddedDBCluster(pxc_cluster_name);
+  });
+
+Scenario('PMM-T452 Verify force unregistering Kubernetes cluster @dbaas @not-pr-pipeline',
+  async ({ I, dbaasPage }) => {
     I.amOnPage(dbaasPage.url);
     dbaasPage.checkCluster(clusterName, false);
-    I.click(dbaasPage.tabs.dbClusterTab.dbClusterTab);
+    dbaasPage.unregisterCluster(clusterName, true);
+    dbaasPage.checkCluster(clusterName, true);
+  });
+
+Scenario('PMM-T524 Delete PXC Cluster and Unregister K8s Cluster @dbaas @not-pr-pipeline',
+  async ({ I, dbaasPage }) => {
+    await dbaasPage.waitForDbClusterTab(clusterName);
     I.waitForElement(dbaasPage.tabs.dbClusterTab.addDbClusterButton, 30);
-    I.click(dbaasPage.tabs.dbClusterTab.addDbClusterButton);
-    I.waitForElement(dbaasPage.tabs.dbClusterTab.basicOptions.fields.clusterNameField, 30);
-    I.fillField(dbaasPage.tabs.dbClusterTab.basicOptions.fields.clusterNameField, pxc_cluster_name);
-    I.click(dbaasPage.tabs.dbClusterTab.basicOptions.fields.kubernetesClusterDropDown);
-    I.waitForElement(
-      dbaasPage.tabs.dbClusterTab.basicOptions.fields.kubernetesClusterDropDownSelect(clusterName),
-      30,
-    );
-    I.click(dbaasPage.tabs.dbClusterTab.basicOptions.fields.kubernetesClusterDropDownSelect(clusterName));
-    I.click(dbaasPage.tabs.dbClusterTab.basicOptions.fields.dbClusterDatabaseTypeField);
-    I.waitForElement(dbaasPage.tabs.dbClusterTab.basicOptions.fields.dbClusterDatabaseTypeFieldSelect('MySQL'));
-    I.click(dbaasPage.tabs.dbClusterTab.basicOptions.fields.dbClusterDatabaseTypeFieldSelect('MySQL'));
+    await dbaasPage.deleteXtraDBCluster(pxc_cluster_name, clusterName);
+  });
+
+Scenario('Single Node PXC Cluster with Custom Resources @dbaas @not-pr-pipeline',
+  async ({ I, dbaasPage }) => {
+    const configuration = {
+      topology: 'Single',
+      numberOfNodes: '1',
+      resourcePerNode: 'Custom',
+      memory: '2 GB',
+      cpu: '2',
+      disk: '5 GB',
+      dbType: 'MySQL',
+      clusterDashboardRedirectionLink: `/graph/d/pxc-cluster-summary/pxc-galera-cluster-summary?var-cluster=${pxc_cluster_name_single}-pxc`,
+    };
+
+    await dbaasPage.waitForDbClusterTab(clusterName);
+    await dbaasPage.createClusterAdvancedOption(clusterName, pxc_cluster_name_single, 'MySQL', configuration);
+    I.click(dbaasPage.tabs.dbClusterTab.createClusterButton);
+    I.waitForText('Processing', 30, dbaasPage.tabs.dbClusterTab.fields.progressBarContent);
+    await dbaasPage.postClusterCreationValidation(pxc_cluster_name_single, clusterName);
+    await dbaasPage.validateClusterDetail(pxc_cluster_name_single, clusterName, configuration);
+    await dbaasPage.deleteXtraDBCluster(pxc_cluster_name_single, clusterName);
+  });
+
+Scenario('PSMDB Cluster with Custom Resources @dbaas @not-pr-pipeline',
```

Comment:
same here about ID

Reply:
Need to check if this works, probably the test will need to be removed, checking with job, if not will add here too

Reply:
Done